perf/bloom_filter/results.yml

```diff
@@ -0,0 +1,355 @@
+builds:
```

Comment:
Please drop this file, artifcats should not be in VCS.

Reply:
Dropped the file. Will post results to the commit message as soon as benchmark is fixed.